Documentation/deploy-master.md

```diff
@@ -88,11 +89,14 @@ The [kubelet](http://kubernetes.io/docs/admin/kubelet.html) is the agent on each
 
 On the master node, the kubelet is configured to communicate with the API server, but not register for cluster work, as shown in the `--register-schedulable=false` line in the YAML excerpt below. This prevents user pods being scheduled on the master nodes, and ensures cluster work is routed only to task-specific worker nodes.
 
+When using Calico, the kubelet is configured to use CNI for networking. This makes Calico aware of each pod that is created and allows it to network the pods into the flannel overlay. The Calico CNI plugin is actually called by the Flannel CNI plugin so that the correct IP range that flannel has selected for the host can be used.
```

Comment:
> The Calico CNI plugin is actually called by the Flannel CNI plugin so that the correct IP range that flannel has selected for the host can be used.

This was a little confusing to me. What if we changed it to something similar to:

"Both flannel and Calico communicate via CNI interfaces to ensure the correct IP range (managed by flannel) is used for each node."

- I swapped host for node since that's used elsewhere in this doc (and others).
- flannel is always lowercase. We had mixed case in this sentence before.